MSAL/src/MSALPublicClientApplication.m

```diff
@@ -988,6 +988,8 @@ - (void)acquireTokenWithParameters:(MSALInteractiveTokenParameters *)parameters
     msidParams.clientCapabilities = self.internalConfig.clientApplicationCapabilities;
     
     msidParams.validateAuthority = _validateAuthority;
+    //TODO: set the following property with instance aware flag in config or exextraURLQueryParameters
```

Comment:
nit: typo on extra query parameter.
also todo: address/decide public header to allow setting instace_aware for requests

Reply:
addressed :)